src/parser/jobs/rpr/modules/Interrupts.tsx

```diff
@@ -8,8 +8,7 @@ export class Interrupts extends CoreInterrupts {
 	static override displayOrder = DISPLAY_ORDER.INTERRUPTS
 
 	override suggestionContent = <Trans id="rpr.interrupts.suggestion.content">
-		Avoid interrupting casts by either prepositioning yourself, utilising slidecasting where possible.
-		If you need to move, consider using <ActionLink action="HELLS_INGRESS"/> or <ActionLink action="HELLS_EGRESS"/>
-		and getting a free <ActionLink action="HARPE"/> out of it.
+		Avoid interrupting casts by either pre-positioning yourself or slidecasting where possible.
+		If you need to move, consider using <ActionLink action="HELLS_INGRESS"/> or <ActionLink action="HELLS_EGRESS"/> and getting a free <ActionLink action="HARPE"/> out of it.
```

Comment:
Nit; Not sure I like the "free Harpe" in this sentence as even if the harpe is free,  Sentence seems to imply harpe should be used in pair with movement when that is not always the case.

Reply:
To match the default suggestion of `Avoid interrupting casts by either prepositioning yourself or utilizing slidecasting where possible. If you have to move, try to save an instant cast to keep your GCD rolling.` you can try:

`If you have to move, consider using <ActionLink action="HELLS_INGRESS"/> or <ActionLink action="HELLS_EGRESS"/> and using the instant cast <ActionLink action="HARPE"/> to keep your GCD rolling.`